### ISSUE TITLE
Cannot run  the script nova.py

### ISSUE THREAD

**Issue by mkarenko** [Dymstro/nova-chatmix-linux#1]
Hi, I cannot run the script, im on fedora 39.
![image](https://github.com/Dymstro/nova-chatmix-linux/assets/71224089/1ccdab00-0029-4273-a31a-0e2fefcdb1da)

**Comment by Dymstro** [comment]
You accidentally downloaded the github page containing the script.
The actual script can be downloaded here https://raw.githubusercontent.com/Dymstro/nova-chatmix-linux/main/nova.py

**Comment by mkarenko** [comment]
Sure, I understand. I've just sorted out that it's my fault. Thanks.

However, I still have another problem. My headphones are Nova 7X, and after finding the card and changing it in the script, plus a few other things, I managed to make the chat/game mix work. The problem I have now is that after running your script, my Bluetooth soundbar no longer shows up in the sound cards. Basically, apart from the Arctis Nova 7X Digital Output, Game Mix, and Chat Mix, nothing else shows up for me. After Bluetooth is connected, but nothing else detects it. I mean the command pactl list sinks.

My question is it possible to have game/chat mix and my soundbar? I gather that I need put into your script a line for my soundbar to be added?

![image](https://github.com/Dymstro/nova-chatmix-linux/assets/71224089/2fbcc98d-b434-4d1f-85c2-6e344aa837f3)
![image](https://github.com/Dymstro/nova-chatmix-linux/assets/71224089/1131cdf7-db43-499f-a959-92b0d081df80)
